[BUG] Regression: Cannot load `darkreader` with Webpack

### **Describe the bug**

As of https://github.com/darkreader/darkreader/pull/4107 I get the following error in my Webpack build:
```
Module not found: Error: Can't resolve 'darkreader' in 'lib/dark-mode.js'
```

### **Steps to reproduce**

- In a Webpack project...
- Integrate `darkreader` as described in [README: Using for a website](https://github.com/darkreader/darkreader#using-for-a-website) using ES Modules
- Run Webpack to create bundled JS containing `darkreader`

### **Expected behavior**

Webpack can find and bundle the `darkreader` module

### **Actual behavior**

Webpack cannot find the `darkreader` module

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "test:utils": "jest --config=tests/utils/jest.config.js",
     "test:utils:debug": "node --inspect-brk ./node_modules/jest/bin/jest --config=tests/utils/jest.config.js --runInBand --no-cache --watch"
   },
-  "browser": "darkreader.js",
+  "main": "darkreader.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/darkreader/darkreader.git"

